### PR TITLE
Update avidemux_sk.ts

### PR DESCRIPTION
**Pull request by jozefM923**
Update slovak translation for avidemux version 2.7.5

### CODE REVIEW

File: avidemux_sk.ts [hunk 1]
```diff
@@ -796,7 +796,7 @@ Vypustiť časové údaje?</translation>
     <message>
         <location line="+3"/>
         <source>Alt+Return</source>
-        <translation>Alt+Return</translation>
+        <translation>Alt+Enter</translation>
```
Comment (eumagga0x2a) [inline avidemux_sk.ts:799]: Thanks, but how did this manage to sneak back in?

File: avidemux_sk.ts [hunk 2]
```diff
@@ -940,7 +940,7 @@ Vypustiť časové údaje?</translation>
     <message>
         <location line="+3"/>
         <source>Ctrl+Return</source>
-        <translation>Ctrl+Return</translation>
+        <translation>Ctrl+Enter</translation>
```
Comment (eumagga0x2a) [inline avidemux_sk.ts:943]: See above. Please fix.

**Comment by jozefM923** [comment]
Hi,
I think the translation is okay. Slovak keyboard contains "Enter" button
and not "Return"
Jozef

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Bez
virů. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

so 18. 4. 2020 o 11:28 eumagga0x2a <notifications@github.com> napísal(a):

**Comment by eumagga0x2a** [comment]
> Hi, I think the translation is okay. Slovak keyboard contains "Enter" button and not "Return"

Have you tested the translation by generating the qm file and running Avidemux in sk_SK.UTF-8 locale with it? In my testing, both affected shortcuts stopped working, independently from selected keyboard layout.

To avoid future issues, I've moved these shortcuts out of reach of lrelease into C++ code, so that this PR can be merged as-is.

Thank you very much for the great work, please keep contributing.

**Pull request merged**